pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -288,14 +288,12 @@ func (ctrl *PersistentVolumeController) shouldDelayBinding(claim *v1.PersistentV
 		return false, nil
 	}
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.DynamicProvisioningScheduling) {
-		// When feature DynamicProvisioningScheduling enabled,
-		// Scheduler signal to the PV controller to start dynamic
-		// provisioning by setting the "annSelectedNode" annotation
-		// in the PVC
-		if _, ok := claim.Annotations[annSelectedNode]; ok {
-			return false, nil
-		}
+	// When feature VolumeScheduling enabled,
+	// Scheduler signal to the PV controller to start dynamic
+	// provisioning by setting the "annSelectedNode" annotation
+	// in the PVC
+	if _, ok := claim.Annotations[annSelectedNode]; ok {
```

Comment:
The annotation needs to move to beta too.  I would actually not prefix the annotation with "beta" either, it will make going to GA easier.